server/store/middleware/infohash/blacklist_test.go

```diff
@@ -40,12 +40,26 @@ var mock store.StringStore = &storeMock{
 	strings: make(map[string]struct{}),
 }
 
+var (
+	ih1, ih2 chihaya.InfoHash
+)
+
+func init() {
```

Comment:
This doesn't need to be in an `init` function. You can use literals on new types derived from arrays: https://play.golang.org/p/s1ISSQ_XMq